src/cell_lists/full_grid.jl

```diff
@@ -31,7 +38,8 @@ struct FullGridCellList{C, LI, MC} <: AbstractCellList
 end
 
 function FullGridCellList(; min_corner, max_corner, search_radius = 0.0,
-                          periodicity = false)
+                          periodicity = false, backend = DynamicVectorOfVectors{Int32},
+                          max_points_per_cell = 100)
```

Comment:
`DynamicVectorOfVectors` is contiguous, GPU-compatible, and supports fully parallel update. That's why it's the default. `Int32` is the default because we can save memory and it's more efficient on GPUs, and we will never have 2 billion particles.

Reply:
Add what to the docs? All kwargs are there.

Reply:
I mean in the constructor
`    FullGridCellList(; min_corner, max_corner, search_radius = 0.0, periodicity = false)`

Reply:
Ah, oops!